vignettes/ar-grid.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Create A&R Grid"
+output:
+  rmarkdown::html_vignette
+resource_files:
+   - package_manual.pdf
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Create A&R Grid}
+---
+
+```{r, echo = FALSE, message = FALSE, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  out.width = "100%"
+)
+```
+
+```{r, message = FALSE}
+library(metalite)
+library(r2rtf)
+library(dplyr)
+```
+
+## Overview
+
+In this document, we illustrate how to create the A&R grid by `metalite`.
+
+## Build a Meta Data
```

Comment:
"data" is a singular uncountable noun, so use either "Build a metadata object" or "Build metadata".

Also, avoid using title case for headings: https://practicaltypography.com/headings.html